tests/clims/services/test_substances.py

```diff
@@ -32,10 +34,13 @@ def csv_sample_submission_path():
 class TestGemstoneSampleSubmission(SubstanceTestCase):
     def setUp(self):
         self.gemstone_sample_type = self.register_gemstone_type()
+        logger = logging.getLogger('clims.files')
```

Comment:
Don't know if logging should be enabled in the tests?

Reply:
This is my trick to get rid of the logging!